.github/workflows/functional-test.yaml

```diff
@@ -694,14 +696,38 @@ jobs:
     runs-on: ubuntu-latest
     if: failure() && github.event_name == 'schedule' && github.repository == 'radius-project/radius'
     steps:
+      - name: Count recently failed tests
+        id: count_failures
+        uses: actions/github-script@v7
+        with:
+          script: |
+            response = await github.rest.actions.listWorkflowRuns({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              workflow_id: 'long-running-azure.yaml',
+              event: 'schedule',
+              per_page: 10
+            });
```

Comment:
Is this going to check for a specific run? Because this seems like there will always be more than one workflow that failed and it is always going to create an issue?

Reply:
> Is this going to check for a specific run? Because this seems like there will always be more than one workflow that failed and it is always going to create an issue?

Yes it will check the specific workflow. Please see L707
```
workflow_id: 'long-running-azure.yaml',
```

Reply:
Ah I have to update it to functional test.